Resources/Locale/en-US/_CD/reagents/RPmeds.ftl

```diff
@@ -95,3 +95,10 @@ reagent-effect-antipsych-normal4 = You space out for a moment.
 reagent-effect-antipsych-normal5 = You feel a bit drowsy.
 
 reagent-effect-antipsych-fade = The anti-psychotic's effects start to fade...
+
+reagent-effect-anti-amnesia-normal1 = You recall things from the past more clearly...
+reagent-effect-anti-amnesia-normal2 = A suppressed memory comes to light...
+reagent-effect-anti-amnesia-normal3 = Your mind feels exceptionally clear...
+reagent-effect-anti-amnesia-normal4 = You remember small thing you forgot.
+
+reagent-effect-anti-amnesia-fade = Your mind no longer feels as clear...
```

Comment:
I am not entirely sold on this text. If you have a better idea please let me know.

Reply:
"You remember small thing you forgot" could be better worded as "You remember something small you forgot"

Reply:
Right... what I wrote is not english.